### PR TITLE
database migration script

### PR DESCRIPTION
**Pull request by MikeLing**
Hey @jmaher ,sorry for the late reply. I'm doing some preparation for beginning of school. 

I had test this script locally and it seems works good to me. But I can't deploy it on heroku directly except merge it into the master/heroku codebase. Please merge this script to master branch first and we could ask @armenzg about if our stage and production server are using same database on heroku.

### CODE REVIEW

File: tools/database_migration.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                               data['failure_classification'], data['failures'])
+            try:
+                session.add(Testjob)
+                session.commit()
+
+            except:
+                session.rollback()
+
+            finally:
+                session.close()
```
Comment (jmaher) [inline tools/database_migration.py]: do we need to close the session each time, or can we leave it open until the while loop is finished?

File: tools/database_migration.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            Testjob = Testjobs(data['slave'], data['result'], data['build_system_type'],
+                               data['duration'], data['platform'], data['buildtype'], data['testtype'],
+                               data['bugid'], data['branch'], data['revision'], data['date'],
+                               data['failure_classification'], data['failures'])
+            try:
+                session.add(Testjob)
+                session.commit()
+
+            except:
+                session.rollback()
```
Comment (jmaher) [inline tools/database_migration.py]: there is no indication that there will be missed data- we need a way to log our progress.

**Comment by jmaher** [comment]
please address my two issues here, then we can get this on seta-dev first, run the script, if it works- we can determine if the DB is the same- It would be nice if they had the same backend data for testjobs, the other data might not be good to share if we change failures.py, etc. it could cause confusion.

**Comment by MikeLing** [comment]
@jmaher r?

**Comment by jmaher** [comment]
merged this- now we need to run this on the heroku commandline?

**Pull request merged**